tests/integration/blocks/test_flow.py

```diff
@@ -47,9 +47,7 @@ def run_block():
                 block_to_run.run_right()
 
             msgQueue.run_lambda(run_block)
-            time.sleep(0.1)
-            while block_to_run.is_running:
-                time.sleep(0.1)  # wait for the execution to finish.
+            time.sleep(2)
```

Comment:
We need to figure out a way to keep the old thing because waiting 2 sec for a test to complete is way to slow.